libpng/plan.sh

```diff
@@ -10,15 +12,15 @@ pkg_build_deps=(core/gcc core/make core/coreutils core/diffutils
                 core/autoconf core/automake)
 pkg_lib_dirs=(lib)
 pkg_include_dirs=(include)
+pkg_bin_dirs=(bin)
 
 do_build() {
   _zlib_dir=$(pkg_path_for zlib)
 
-  export CPPFLAGS="${CPPFLAGS} ${CFLAGS}"
-  ./configure --prefix=${pkg_prefix} \
+  ./configure --prefix="${pkg_prefix}" \
               --host=x86_64-unknown-linux-gnu \
               --build=x86_64-unknown-linux-gnu \
               --disable-static \
-              --with-zlib-prefix=${_zlib_dir} \ &&
+              --with-zlib-prefix="${_zlib_dir}" \ &&
   make && make install
```

Comment:
Not part of your PR, but any reason we are doing `make install` as part of the build?